### ISSUE TITLE
ProForma notation file format

### ISSUE THREAD

**Issue by veitveit** [topdownproteomics/proteoform-nomenclature-standard#11]
Files containing sequences that are annotated according to the nomenclature:

_Suggestions_:
Extension: .ptm.fasta or just .proteoforms
Format name: Proteoform format, PTM format

**Comment by trishorts** [comment]
.pf .pfm .pForm

**Comment by sgibb** [comment]
PTM often stands for post translational modification so it is maybe misleading.

`.ptm.fasta` would be ok, `.proteoforms` is very long.

What about `.ptf` or `.ptf.fasta`?

**Comment by stefanks** [comment]
Yes, it would be great to have a file format for storing proteoforms: it could be used downstream for proteoform database searches, etc.

Each fasta entry has a header and a sequence. I believe that the only requirement for a header is that it start with a > character. 

I propose the following rules for our files:
1) Header starting with > character
2) Every header is unique in a file, so it could act as a key
3) Sequence in our format

A complete specification doesn't need much more than this!

I like .pf .ptf .pfm

**Comment by rfellers** [comment]
I like the shorter file extensions without .fasta, but don't have a favorite. Should we allow line breaks in the middle of a ProForma string like a typical FASTA file does?
